.github/workflows/test-matrix.yml

```diff
@@ -5,24 +5,16 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, macos-latest]
-        contract: [assemblyscript, rust]
-        frontend: ['vanilla', 'react', 'none']
```

Comment:
Tests are separate now to 2 workflows:
- test-matrix: tests generation of all templates and compares to their saved snapshots (see `test/make.test.ts`)
- test-e2e: generates templates, npm installs, and runs contract tests (`test/e2e.sh` - this works only on linux in github-actions, for some reason on macos it times-out

currently e2e break because of this: https://github.com/near/near-cli/issues/796
the CLI expects an input on first run